resources/views/templates/profile/form.blade.php

```diff
@@ -1,72 +1,83 @@
-                        @foreach($columns as $col)@if($col['name'] == 'password')<div class="form-group row align-items-center" :class="{'has-danger': errors.has('{{ $col['name'] }}'), 'has-success': this.fields.{{ $col['name'] }} && this.fields.{{ $col['name'] }}.valid }">
```

Comment:
The resulted formatting is not as expected, however I do not know how to change it, to have exported files formatted correctly.